04-onion-routing.md

```diff
@@ -208,12 +209,32 @@ This is formatted according to the Type-Length-Value format defined in [BOLT #1]
     1. type: 12 (`current_path_key`)
     2. data:
         * [`point`:`path_key`]
+    1. type: 14 (`outgoing_node_id`)
```

Comment:
Linking https://github.com/lightning/bolts/pull/814 again. Perhaps the intermediate step helps making this PR easier to get merged?

Reply:
Easier in that things can be implemented and deployed in stages. First do pubkey routing and then continue with trampoline building on top of that.

Reply:
I just noticed the subset and that it could be an intermediate step. But not too certain that it is useful either.

The trampoline spec, does that include pubkey routing between non-trampoline routes too though?

Reply:
This PR adds tlv 14 `outgoing_node_id` to the spec. Does that mean that pubkey routing can then also be used in regular non-trampoline forwards going through the node implementing this spec?

Reply:
No further comments. You are right :)